### PR TITLE
Updates to 1.9.0 release notes

### PR DESCRIPTION
**Pull request by mikewalch**
This PR does not publish the release notes or make them complete.  More work needs to be done.

### CODE REVIEW

**Comment by keith-turner** [comment]
Whats there looks good.  I would like to make some changes when you are finished.

File: _posts/release/2018-03-06-accumulo-1.9.0.md [hunk 1]
```diff
@@ -3,10 +3,226 @@ title: Apache Accumulo 1.9.0
 draft: true
 ---
 
+Apache Accumulo 1.9.0 is a minor release on the 1.x branch. This release would be considered a maintainence release on 1.8
```
[the remark this answers is not in the document]
Comment (mikewalch) [reply _posts/release/2018-03-06-accumulo-1.9.0.md]: fixed.  see b004a9c54306e

File: _posts/release/2018-03-06-accumulo-1.9.0.md [hunk 2]
```diff
@@ -3,10 +3,226 @@ title: Apache Accumulo 1.9.0
...
+branch except there are some API additions and performance improvements which resulted in a new minor release. Users of 1.8.x
```
[the remark this answers is not in the document]
Comment (mikewalch) [reply _posts/release/2018-03-06-accumulo-1.9.0.md]: removed the text. see b004a9c54306e

File: _posts/release/2018-03-06-accumulo-1.9.0.md [hunk 3]
```diff
@@ -3,10 +3,226 @@ title: Apache Accumulo 1.9.0
...
+versions of Accumulo should upgrade to 1.9.0. There will be no more bug fix releases on the 1.8 branch. This release contains
+changes from 93 issues. [A list of all changes](#all-changes) in this release is available at the bottom of this page.
```
[the remark this answers is not in the document]
Comment (mikewalch) [reply _posts/release/2018-03-06-accumulo-1.9.0.md]: I left all changes in until the release notes are finished but I marked that they should be removed.

**Pull request merged**